apis/shopper-stores/shopper-stores.raml

```diff
@@ -0,0 +1,98 @@
+#%RAML 1.0
+title: Shopper Stores
```

Comment:
With the SDKs, we usually don't add/update APIs manually but through the `updateApis` command. We can add this API manually, but for future updates, we need to ensure that `shopper-stores` is added to the API list so that we can grab API updates from anypoint exchange:

https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/d336d21af6924d60f1a81526f1c87f78d5d024f3/scripts/config.ts#L14

Also, seems like the CI is failing due to the increased bundle size, if you pull in the latest changes to main, we updated the bundle size limit in this PR: https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/pull/150

Reply:
@jeremy-jung1 there's some overlap in this other PR #153, where we're pulling the latest API changes that also include the shopper-stores.